pkg/kubecli/kubecli.go

```diff
@@ -66,12 +70,39 @@ func GetKubevirtClientFromFlags(master string, kubeconfig string) (KubevirtClien
 	return &kubevirt{master, kubeconfig, restClient, coreClient}, nil
 }
 
+func GetKubevirtClientFromFlags(flags *flag.FlagSet) (KubevirtClient, error) {
```

Comment:
this function isn't used anywhere. If we remove it, then the GetKubevirtClientFromConfig function can remain as GetKubevirtClientFromFlags and this patch becomes much simpler.

Reply:
We currently have a few modules in https://github.com/kubevirt/kubevirt/pull/817 that are all duplicating the same code. It makes more sense to add it here as its a general function (and can't go in the other PR due to the server/master conflict here)--but I do plan to put this function to immediate use if/when accepted. :)